namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-query-development/resources/pingdom.tf

```diff
@@ -0,0 +1,26 @@
+################################################################################
+# Track a Query (Correspondence Tool Staff)
+# Pingdom check for uptime monitoring
+#################################################################################
+
+terraform {
+  backend "s3" {}
+}
```

Comment:
In terraform 0.12 we don't need backend here, if you already have a backend in main.tf, can you please remove it

please see this example
`https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/cccd-production/resources/pingdom.tf`

Reply:
The official document shows this:
https://user-guide.cloud-platform.service.justice.gov.uk/tasks.html#creating-pingdom-checks-prerequisites